common/src/main/java/org/apache/cassandra/sidecar/common/CassandraAdapterDelegate.java

```diff
@@ -50,7 +50,7 @@ public class CassandraAdapterDelegate implements ICassandraAdapter, Host.StateLi
     private volatile Session session;
     private SimpleCassandraVersion currentVersion;
     private ICassandraAdapter adapter;
-    private volatile boolean isUp = false;
+    private volatile NodeSettings status = null;
```

Comment:
Can the status be renamed to settings or nodeSettings? Doesn't it make sense to keep both of them (isUP and nodeSettings)?